app/query_objects/cafe_queries.rb

```diff
@@ -0,0 +1,44 @@
+class CafeQueries
+  # returns: all users who worked during study period(lp) and year
+  def self.working_users(lp, year)
+    join_cafe_shifts(User.joins(:cafe_shifts), lp, year).includes(:cafe_workers).uniq
+  end
+
+  # returns: all cafe_workers from study period and year
+  def self.cafe_workers(lp, year)
+    join_cafe_shifts(CafeWorker.joins(:cafe_shift), lp, year)
+  end
+
+  def self.highscore(lp, year)
+    join_cafe_shifts(User.select('users.id, users.firstname, users.lastname, count(*) as score')
+                     .joins(:cafe_shifts), lp, year)
+    .group('user_id')
+    .order('score desc')
+    .limit(10)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.